linux/just_files/new_just

```diff
@@ -585,34 +712,35 @@ function write_dockerfile()
             #T# 4) just sync # (optionally updates the docker image)
             #T# 5) just shell
             #T# 6) python -c "import osgeo.gdal_array" # Should not create an error message
-
-            ARG SKIP_PIPENV_SYNC
-            #T# Sometimes you need to fix the image so that pipenv lock can be updated, but
-            #T# you can'"'"'t because pipenv sync won'"'"'t work. This chicken/egg cycle can be
-            #T# escaped by setting the build arg SKIP_PIPENV_SYNC to 1. Then, pipenv sync is
+            #T#
+            #T# Call pip-fake for each editable package
+            RUN pip-fake "'"$(docker_add_quote_escape "/src/${RELATIVE_PATH}")"'" vsi python/vsi; \
```

Comment:
I don't want it first (as that's the `RUN` line) or last (since the last line can't end in `; \`). But putting it in the middle makes is so when it is uncommented, it is syntactically correct. It only shows up with tutorial mode on. So not that big of a deal. I went ahead and changed the order so `/src` comes first.